netty/src/main/java/io/grpc/netty/HandlerSettings.java

```diff
@@ -36,7 +36,7 @@
 
 /**
  * Allows autoFlowControl to be turned on and off from interop testing and flow control windows to
- * be accessed.
+ * be accessed. For internal testing only.
  */
 @VisibleForTesting
```

Comment:
Should not add `For internal testing only.`
Should add comment after `@VisibleForTesting`
`@VisibleForTesting // Visible for tests in other packages.`